From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Sat, 31 Jan 2026 23:58:45 +0100
Subject: [PATCH] HACK: lower required rust version

1.92 is not actually needed and this allows building with
OSELAS.Toolchain-2025.11.1.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 Cargo.toml                                                              | 2 +-
 .../workspaces/gstreamer-rs-0.25.3+4e20c0caa010/Cargo.toml              | 2 +-
 .../workspaces/gtk-rs-core-0.22.8+ae5dabb51ced/Cargo.toml               | 2 +-
 ptxdist-cargo-home/workspaces/gtk4-rs-0.11.3+6e427d7afd0e/Cargo.toml    | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index 43e40cca351f..6368344f9c05 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -151,7 +151,7 @@ lto = "off"
 version = "0.15.3"
 repository = "https://gitlab.freedesktop.org/gstreamer/gst-plugins-rs"
 edition = "2024"
-rust-version = "1.92"
+rust-version = "1.91"
 
 [workspace.dependencies]
 glib = { git = "https://github.com/gtk-rs/gtk-rs-core", branch = "0.22", version = "0.22.0" }
diff --git a/ptxdist-cargo-home/workspaces/gstreamer-rs-0.25.3+4e20c0caa010/Cargo.toml b/ptxdist-cargo-home/workspaces/gstreamer-rs-0.25.3+4e20c0caa010/Cargo.toml
index 61e1dec3135f..eb9131107f26 100644
--- a/ptxdist-cargo-home/workspaces/gstreamer-rs-0.25.3+4e20c0caa010/Cargo.toml
+++ b/ptxdist-cargo-home/workspaces/gstreamer-rs-0.25.3+4e20c0caa010/Cargo.toml
@@ -124,7 +124,7 @@ categories = ["api-bindings", "multimedia"]
 repository = "https://gitlab.freedesktop.org/gstreamer/gstreamer-rs"
 homepage = "https://gstreamer.freedesktop.org"
 edition = "2024"
-rust-version = "1.92"
+rust-version = "1.91"
 
 [workspace.dependencies]
 gio = { git = "https://github.com/gtk-rs/gtk-rs-core", branch = "0.22", version = "0.22.0" }
diff --git a/ptxdist-cargo-home/workspaces/gtk-rs-core-0.22.8+ae5dabb51ced/Cargo.toml b/ptxdist-cargo-home/workspaces/gtk-rs-core-0.22.8+ae5dabb51ced/Cargo.toml
index 215e3a98318e..8afc98ba7ab8 100644
--- a/ptxdist-cargo-home/workspaces/gtk-rs-core-0.22.8+ae5dabb51ced/Cargo.toml
+++ b/ptxdist-cargo-home/workspaces/gtk-rs-core-0.22.8+ae5dabb51ced/Cargo.toml
@@ -65,7 +65,7 @@ repository = "https://github.com/gtk-rs/gtk-rs-core"
 license = "MIT"
 exclude = ["gir-files/*"]
 edition = "2024"
-rust-version = "1.92"
+rust-version = "1.91"
 
 [workspace.dependencies]
 libc = "0.2"
diff --git a/ptxdist-cargo-home/workspaces/gtk4-rs-0.11.3+6e427d7afd0e/Cargo.toml b/ptxdist-cargo-home/workspaces/gtk4-rs-0.11.3+6e427d7afd0e/Cargo.toml
index 4004aba2150a..f18b4512b9f8 100644
--- a/ptxdist-cargo-home/workspaces/gtk4-rs-0.11.3+6e427d7afd0e/Cargo.toml
+++ b/ptxdist-cargo-home/workspaces/gtk4-rs-0.11.3+6e427d7afd0e/Cargo.toml
@@ -41,7 +41,7 @@ edition = "2024"
 homepage = "https://gtk-rs.org/gtk4-rs"
 license = "MIT"
 repository = "https://github.com/gtk-rs/gtk4-rs"
-rust-version = "1.92"
+rust-version = "1.91"
 version = "0.11.3"
 
 [workspace.dependencies]
